tests/resources/test_resources.py

```diff
@@ -727,6 +727,39 @@ def test_link_update(
     assert link_result.json["permission"] == "preview"
 
 
+def test_response_handlers(running_app, minimal_record, client_with_login):
+    # Check that the response handlers are correctly configured.
+    # The correctness of its serialization is tested in /tests/serializers
+
+    accept = [
+        "application/json",
+        # "application/marcxml+xml",  # Fails on search (not implemented)
```

Comment:
major: the purpose of this test is to showcase that we are missing serialization, and they are not being thoroughly tested in the serializer tests.